packages/transport/test/modules/sync/externalEventProcessors/PeerDeletedExternalEventProcessor.test.ts

```diff
@@ -35,8 +35,17 @@ describe("PeerDeletedExternalEventProcessor", function () {
 
     test("PeerDeletedExternalEventProcessor should mark peer as deleted", async function () {
         const eventProcessor = new PeerDeletedExternalEventProcessor(recipient.identityDeletionProcess.eventBus, recipient);
-        await eventProcessor.execute({ id: "anId", createdAt: "aDate", index: 1, syncErrorCount: 0, type: "PeerDeleted", payload: { relationshipId: relationshipId.toString() } });
+        const deletionDate = CoreDate.local();
+        await eventProcessor.execute({
+            id: "anId",
+            createdAt: "aDate",
+            index: 1,
+            syncErrorCount: 0,
+            type: "PeerDeleted",
+            payload: { relationshipId: relationshipId.toString(), deletionDate: deletionDate }
```

Comment:
sorry, I didn't saw this. 

```suggestion
            payload: { relationshipId: relationshipId.toString(), deletionDate }
```

we always use the shorthand if possible.